.ci/obfuscate/gnirts.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+const gnirts = require('gnirts');
+const fs = require('fs')
+
+const rootDir = process.cwd()
+
+const obfuscateJSFiles = () => {
+  const dirFilesToObfuscate = [ 'lib/hooks/analytics/analytics.js' ]
```

Comment:
Sorry, do we need an array for only one file? And name `dirFilesToObfuscate` looks strange, because we have variable to store array of the files and none of them isn't a directory.